lib-ui-core/src/main/java/com/pax/us/pay/ui/core/helper/EnterTipHelper.java

```diff
@@ -32,9 +32,13 @@ public void sendNext(long amount) {
         super.sendNext(bundle);
     }
 
-    public void sendNext() {
+    public void sendNext(Long amount) {
         Bundle bundle = new Bundle();
-        bundle.putBoolean(EntryRequest.PARAM_NO_TIP_SELECTED, true);
+        if (null == amount )
+            bundle.putBoolean(EntryRequest.PARAM_NO_TIP_SELECTED, true);
```

Comment:
Can we just set `EntryRequest.PARAM_TIP` to 0?

Reply:
And when BYPASS, don't return EntryRequest.PARAM_TIP?
Ticket:
https://pax-us.atlassian.net/browse/BPOSANDJAX-517

Reply:
Yes, change `PARAM_TIP` to optional to handle BYPASS.
And PLEASE make sure user can get enough information here without reviewing our private repo.

Reply:
Done https://pax-us.atlassian.net/browse/ADJ-199

Reply:
3c1453c002e2c72f7ef98129787655cf0e798d4b
7e9605bee8f07ab8d1a1d657aae0cf3e19164f6a

The most important thing needs to be done is 
UPDATE the javadoc  `* request: {@link EntryRequest#PARAM_TIP} <br>`.

Reply:
Already done on https://github.com/PAXTechnologyInc/POSLink-UI/pull/74